microprofile-jwt/pom.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>microprofile-test-suite</artifactId>
+        <groupId>org.jboss.eap.qe</groupId>
+        <version>1.0.0.Final-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>microprofile-jwt</artifactId>
+
+    <dependencies>
```

Comment:
Why are these deps needed here ?

Why aren't deps in root pom.xml enough ?
https://github.com/jboss-eap-qe/eap-microprofile-test-suite/blob/4b1b16c1e96f601becbef97b9e71b2a4971bfdab/pom.xml#L37-L67

I don't like explosions of dependencies and their dependencies

Note: Hard-coded versions are hell, using properties with version. prefix is the way to go
Anyway, I don't like those deps, please eliminate them

Reply:
Thank you - I agree with you. Dependencies explosion isn't a good idea and when we finally introduced the `dependencyManagement` there is nothing to talk about. I will move these dependencies to root `pom.xml` (apart those I selected from `dependencyManagement`).